packages/cryptoassets/src/currencies.ts

```diff
@@ -78,6 +78,25 @@ const ethereumUnits = (name, code) => [
 ];
 
 const cryptocurrenciesById: Record<string, CryptoCurrency> = {
+  near: {
+    type: "CryptoCurrency",
+    id: "near",
+    coinType: 0, //TODO: what should I put here?
+    name: "NEAR", //TODO: is it the name of the company, or name of the cryptocurrency?
+    managerAppName: "NEAR",
+    ticker: "N",
+    scheme: "near", //TODO: where this will be used? `near` is ok?
+    color: "#000", //TODO: where this collor woll be used?
+    family: "near", //TODO: where this will be used? `near` is ok?
+    units: [
+      {
+        name: "N",
+        code: "N",
+        magnitude: 8, //TODO: what is the magnitude? How is it used?
```

Comment:
this is very important to get it right when we plan an integration: it represent the pow of 10 of the most minimal unit of the currency.

For instance, in Bitcoin, it's 8 because 1 BTC = 10^8 satoshi.
as we would use "integers" to represent the smallest divisible amount, it needs to be representing this.

for Ethereum as an other example, it's 18.

Reply:
Ok, the smallest one is yN (yocto NEAR). So it should be 24.